test/__tests__/solidity/gatherSources.spec.ts

```diff
@@ -3,7 +3,9 @@ import { vol } from "memfs";
 import { gatherSources, ImportFile, ResolverEngine, SolidityImportResolver } from "../../../src";
 import deepequal = require("deep-equal");
 
-function expectedOutput(filesObj: { [s: string]: string }): ImportFile[] {
+type dictStringString = { [s: string]: string };
```

Comment:
`type Map<K = string, V = string> = { [s: K]: V };`

Reply:
I don't understand what this solves. I just took out the type used in the code like three times and gave it some name.

Reply:
Names a reason why this is used instead of repeating the declaration itself
```
// this line adds two numbers and sets it to A
let A = 2 + 5;
```
map is easily understood data type, dictstringstring not all that much

Reply:
Map in JS implies object with methods like .get(), .set() etc. This is more like Pythonic dictionary, IMHO. I'll go with type name of just 'dictionary'. M'kay?